### DISCUSSION TITLE
Add toggle for permanently expanding the in-game leaderboard without having to hold down a key

### DISCUSSION THREAD

**Discussion by ikkue** [ppy/osu#20633] [Ideas]
To me, the point of having an in-game leaderboard is to be able to quickly look and see what my score, accuracy, and/or highest combo is to the player/score directly above me. That is why I made #20532, which got fixed by #20547, in the first place. I was kind of fine with it when it was implemented for multiplayer in #18221, but I think where you are on the leaderboard ranking-wise does not matter as much in solo as it is in multiplayer.

My suggestion for fixing/implementing this is to revert #18221 and replace holding-for-HUD with a toggle that retracts or expands the leaderboard permanently instead, something like what `TAB` does in stable. I acknowledge that customizability for each element of the HUD is planned eventually, but this is just a temporary solution before then. There also may be a better way of doing this, so if anybody finds this idea feasible as a temporary solution, I'm sure you can come up with a better method as well.

**Comment by peppy** [comment]
Rather than adding any more temporary solutions to this, I would rather implement the proper support required in the layout editor to make this happen.

I will probably get to that this week.
